packages/core-components/src/components/EmptyState/MissingAnnotationEmptyState.tsx

```diff
@@ -77,13 +82,16 @@ export function MissingAnnotationEmptyState(props: Props) {
             highlighted example below:
           </Typography>
           <div className={classes.code}>
-            <CodeSnippet
-              text={COMPONENT_YAML.replace('ANNOTATION', annotation)}
-              language="yaml"
-              showLineNumbers
-              highlightedNumbers={[6, 7]}
-              customStyle={{ background: 'inherit', fontSize: '115%' }}
-            />
+            {annotationsToDisplay.map(ann => (
+              <CodeSnippet
+                key={ann}
+                text={COMPONENT_YAML.replace('ANNOTATION', ann)}
```

Comment:
This can probably also be done differently. Instead of three code snippets that occupy several screens of space, you could change it so that it replaces with several lines in a single snippet :)